advanced03-AF_XDP/af_xdp_kern.c

```diff
@@ -18,6 +19,17 @@ struct bpf_map_def SEC("maps") xdp_stats_map = {
 	.max_entries = 64,
 };
 
+#if LINUX_VERSION_CODE < KERNEL_VERSION(5,3,0)
+
+/* Kernel version before 5.3 needed an additional map */
+struct bpf_map_def SEC("maps") qidconf_map = {
```

Comment:
I don't see this map referred to anywhere; how is it actually used on those old kernels?

Reply:
Think this is part of the older libbpf installing int